db/migrate/20190408143457_add_lat_and_lng_to_schools.rb

```diff
@@ -0,0 +1,7 @@
+class AddLatAndLngToSchools < ActiveRecord::Migration[5.2]
+  def change
+    add_column :schools, :latitude, :float
+    add_column :schools, :longitude, :float
+    remove_column :schools, :geolocation
```

Comment:
Do we need a task to migrate the data over, too?

Reply:
I think (@tahb, correct me if I'm wrong) we should be able to run the `data:schools:import` job on deploy, which will update the schools from the source data. The only issue is there may be a small window where location searches may not work properly.